tempesta_fw/http_msg.c

```diff
@@ -119,7 +119,8 @@ __hdr_is_singular(const TfwStr *hdr)
 {
 	int i, fc;
 	static const TfwStr hdr_singular[] __read_mostly = {
-#define TfwStr_string(v) { (v), NULL, sizeof(v) - 1, 0 }
+#define TfwStr_string(v) { .data = (v), .skb = NULL, .len = sizeof(v) - 1,\
+			   .flags = 0 }
```

Comment:
Frankly, this looks like something that should be placed in `str.h`. In fact, can't `TFW_STR_FROM()` be used for this purpose?